django_components/utils/autodiscover.py

```diff
@@ -0,0 +1,32 @@
+import glob
+from pathlib import Path
+from typing import List, Optional
+
+from django.template.engine import Engine
+
+from django_components.template_loader import Loader
+
+
+def search(search_glob: Optional[str] = None, engine: Optional[Engine] = None):
```

Comment:
I managed to use the autodiscovery logic, thanks @dylanjcastillo for suggestion! I moved the logic to separate file, and refactored it to make glob search optional, so it can used for both use cases.